gateways/js/test/spec/fjage.spec.js

```diff
@@ -335,6 +335,56 @@ describe('A Gateway', function () {
     gw.close();
   });
 
+  it('should be able to request for all agents from the master container', async function() {
+    const gw = new Gateway(gwOpts);
+    await delay(300);
```

Comment:
Why are we waiting here? Opening a gateway and immediately start fetching data is much more common, no?

Reply:
I'm also not sure why we're having a close delay in most tests, but I'm happy to punt that for another day.